src/main/java/htsjdk/tribble/readers/AsciiLineReader.java

```diff
@@ -93,6 +105,10 @@ public final String readLine(final PositionalBufferedStream stream) throws IOExc
             if (c == LINEFEED || c == CARRIAGE_RETURN) {
                 if (c == CARRIAGE_RETURN && stream.peek() == LINEFEED) {
                     stream.read(); // <= skip the trailing \n in case of \r\n termination
+                    this.lineTerminatorLength = 2;
+                }
+                else {
```

Comment:
I don't know about "used to be" I was trained by @nh13 to always have `} else {` I'm not super picky, but I thought it appropriate due to the subject matter.